src/components/Profile.jsx

```diff
@@ -26,7 +26,7 @@ const Profile = (props) => {
       <ArtCard>
         <UserInfo>
           <CardBackground />
-          <Link to={<Profile/>}>
+          <Link to={<Profile/>} className="welcome">
```

Comment:
import { WelcomeLink } from "..."

<WelcomeLink to={<Profile/>} >

I just noticed here I have <Profile/>